spec/components/provider_interface/offer_summary_component_spec.rb

```diff
@@ -126,7 +126,7 @@ def row_link_selector(row_number)
   it 'renders the new course option details' do
     expect(row_text_selector(:provider, render)).to include(course_option.provider.name)
     expect(row_text_selector(:course, render)).to include(course_option.course.name_and_code)
-    expect(row_text_selector(:location, render)).to include(course_option.site.name_and_address)
+    expect(row_text_selector(:location, render)).to include(course_option.site.full_address.gsub(', ', "\n"))
```

Comment:
could we avoid the gsub throughout the tests and pass in the `join_by` attribute instead to the method?

Reply:
yeah woops - this slipped through the find and replace